### DISCUSSION TITLE
On user creation trigger not working locally.

### DISCUSSION THREAD

**Discussion by jaocfilho** [supabase/supabase#14918] [Questions] [answered]
So I've been using supabase with my app prototype for some weeks and now I decided to develop a local version.
I followed all the instructions from the docs and everything worked thine except for my trigger on auth.users table.

I've created a trigger like the docs suggests, that every time an user is created, a public.profiles row for that user is created, and it worked fine on production.
I don't know why, but the initial migration from the remote db didn't worked with this trigger.

I'm not sure if this is intended and I couldn't find help anywhere.
I've been think in create the same trigger locally, but I'm afraid this messes things up the next time I dump my local changes to the remote db.

**Comment by GaryAustin1** [comment, marked as answer]
The migration does not include the auth schema which is where the trigger is.  The function should have been copied over.

**Comment by attrove-wolfe** [comment]
See this post for an actual explanation. 

https://github.com/supabase/cli/issues/1726
